src/app/components/task/Tasks.js

```diff
@@ -222,12 +222,7 @@ const Tasks = ({
     }
     return (
       <div className={inputStyles['form-footer-actions-context']}>
-        Saved {moment(latestEditInfo.latestDate).fromNow()} by{' '}
-        <a
-          href={`/check/user/${latestEditInfo.latestAuthorDbid}`}
-        >
-          {latestEditInfo.latestAuthorName}
-        </a>
+        {`Saved ${moment(latestEditInfo.latestDate).fromNow()} by ${latestEditInfo.latestAuthorName}`}
```

Comment:
This string also must be translated. Ideally this has the same string id as the other instance.

Reply:
done :white_check_mark: